app/views/dashboards/_table_content.html.erb

```diff
@@ -1,6 +1,6 @@
 <!-- locals: title, table_type (for div id and tbody div id), pregnancies, sortable, autosortable -->
 
-<div id="<%= table_type %>" class="margin-bottom mt-5">
+<div id="<%= table_type %>" class="margin-bottom mt-4">
```

Comment:
lots of changes like this in the html, where we gently crank down the margins so there's nt a lot of dead whitespace.